pkg/config/config.go

```diff
@@ -3,16 +3,18 @@ package config
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/pivotal-cf/brokerapi"
 	"io/ioutil"
 )
 
-type BlockheadConfig struct {
-	Password string `yaml:"password,omitempty"`
-	Username string `yaml:"username,omitempty"`
+type Config struct {
+	Password string            `json:"password,omitempty"`
+	Username string            `json:"username,omitempty"`
+	Service  brokerapi.Service `json:"service"`
```

Comment:
This should be plural.

Reply:
why? I thought about making it plural but for the beginning we are going to only support eth in dev mode and in the best case with some plans for it. why do we need to make it more generic than that where we dont need to atm.

Reply:
Because we know we want to support Fabric in the nearish future. If we had no plans I would agree but since we do have one making it work for multiple services makes more sense to me. This also ties into having the array of services in running the broker.

Reply:
point taken. A broker with multiple services probably makes for a good demo, but practically speaking for the case of Fabric I wonder if a single broker would be used to offer both the eth service and the fabric service. 

Anyhow making it a slice sounds good.